lib/datasource/remote/api/tokenmodels_repository.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:async/async.dart';
+import 'package:http/http.dart' as http;
+import 'package:seeds/datasource/remote/api/http_repo/http_repository.dart';
+import 'package:seeds/datasource/remote/api/http_repo/seeds_scopes.dart';
+import 'package:seeds/datasource/remote/api/http_repo/seeds_tables.dart';
+
+/// Retrieve token metadata (used for display of currency cards etc) from
+/// a table in the token master smart contract (tmastr.seeds)
+class TokenModelsRepository extends HttpRepository {
+  Future<Result<List<int>>> getAcceptedTokenIds(String useCase) async {
+    final request = createRequest(
+      code: SeedsCode.accountTokenModels,
+      scope: useCase,
+      table: SeedsTable.tableTokenMasterAcceptances,
+      limit: 100,
+    );
+    return http
```

Comment:
There is no guarantee that this will fetch all tokens. Limit 100 means it won't return more than 100, but it could also just return 3. 

So we need to load the token list until more=false.

Reply:
Note: the APIs have been pretty good about this, but I don't think we can rely on that. 

Also if there's more than 100 tokens in the list?

Reply:
fixed by [3500859](https://github.com/JoinSEEDS/seeds_light_wallet/pull/1880/commits/3500859b1d0c1a782c4c6484e34db7cafbb38e48)